payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -735,6 +750,21 @@ constructor(
         }
     }
 
+    @Parcelize
+    enum class AllowRedisplay(val value: String) : StripeModel {
```

Comment:
Should `value` be internal?

Reply:
Was trying to be consistent with our other enum types which expose `value`. Can make it internal since the string value isn't relevant to the merchant.